src/common/buttons/button.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { ReactNode } from 'react';
+import styles from './button.module.scss';
+
+const DefaultProps = {
+  onClick: undefined,
+  isDisabled: undefined,
```

Comment:
I find this pattern odd but I know we discussed it in a call. @facu274 do you know if this is the proper way to add default props to optional in the template?